@xen-orchestra/web/src/components/account-menu/AccountMenu.vue

```diff
@@ -8,38 +8,83 @@
         @click="open($event)"
       />
     </template>
-    <MenuItem icon="fa:book">
+    <MenuItem class="icon" icon="fa:circle-user">
+      <a class="link typo-body-bold-small" href="/#/user" rel="noopener noreferrer" target="_blank">
+        {{ t('account') }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
+      </a>
+    </MenuItem>
+
+    <MenuItem class="icon" icon="fa:satellite">
+      <a class="link typo-body-bold-small" href="/#/xoa/licenses" rel="noopener noreferrer" target="_blank">
+        {{ t('license-name', { name: 'XOA' }) }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
+      </a>
+    </MenuItem>
+
+    <MenuItem class="icon" icon="fa:book">
+      <a
+        class="link typo-body-bold-small"
+        href="https://docs.xen-orchestra.com"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        {{ t('documentation-name', { name: 'XO' }) }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
+      </a>
+    </MenuItem>
+
+    <MenuItem class="icon" icon="fa:book">
       <a
         class="link typo-body-bold-small"
         href="https://docs.xcp-ng.org?utm_campaign=xo6&utm_term=xcpdoc"
         rel="noopener noreferrer"
         target="_blank"
       >
         {{ t('documentation-name', { name: 'XCP-ng' }) }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
       </a>
     </MenuItem>
-    <MenuItem icon="fa:headset">
+
+    <MenuItem class="icon" icon="fa:headset">
       <a
         class="link typo-body-bold-small"
         href="https://vates.tech/pricing-and-support?utm_campaign=xo6&utm_term=pricing"
         rel="noopener noreferrer"
         target="_blank"
       >
         {{ t('professional-support') }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
       </a>
     </MenuItem>
-    <MenuItem icon="fa:code">
+
+    <MenuItem class="icon" icon="fa:code">
       <a class="link typo-body-bold-small" href="/rest/v0/docs/#/" rel="noopener noreferrer" target="_blank">
         {{ t('documentation-name', { name: 'REST API' }) }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
       </a>
     </MenuItem>
-    <MenuItem icon="fa:arrow-right-from-bracket" class="logout" @click="logout()">
-      {{ t('log-out') }}
+
+    <MenuItem class="icon" icon="fa:message">
+      <a
+        class="link typo-body-bold-small"
+        href="https://xcp-ng.org/forum/category/12/xen-orchestra"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        {{ t('send-us-feedback') }}
+        <VtsIcon size="small" name="fa:arrow-up-right-from-square" />
+      </a>
+    </MenuItem>
+
+    <MenuItem class="icon" icon="fa:arrow-right-from-bracket" @click="logout()">
+      <span class="link typo-body-bold-small">{{ t('log-out') }}</span>
     </MenuItem>
```

Comment:
```suggestion
 <MenuItem v-for="link in links" :key="link.href" class="icon" :icon="link.icon">
      <UiLink size="small" class="link typo-body-bold-small" :href="link.href">
        {{ link.label }}
      </UiLink>
 </MenuItem>
    
 <MenuItem class="icon" icon="fa:arrow-right-from-bracket" @click="logout()">
      <span class="link typo-body-bold-small">{{ t('log-out') }}</span>
 </MenuItem>
```

I think you can do something like that for more readability